package.json

```diff
@@ -27,6 +27,7 @@
     "test": "./node_modules/karma/bin/karma start --single-run"
   },
   "devDependencies": {
+    "babel-cli": "^6.18.0",
```

Comment:
@aossowski no problem, was able to replicate. I must've done `nom install -g babel-cli` as some point on my old machine and my new machine failed immediately. Thanks!